code-samples/popular-topics/permissions/index.js

```diff
@@ -0,0 +1,125 @@
+const util = require('util');
+const { Client, Permissions } = require('discord.js');
+const client = new Client();
+
+client.on('ready', () => {
+	console.log('Ready!');
+});
+
+client.on('message', message => {
+	if (message.author.bot || !message.content.startsWith('!')) return;
+	if (!message.channel.permissionsFor(client.user).has('SEND_MESSAGES')) {
+		return;
+	}
+	const botPerms = ['MANAGE_MESSAGES', 'KICK_MEMBERS', 'MANAGE_ROLES', 'MANAGE_CHANNELS'];
+	if (!message.guild.me.permissions.has(botPerms)) {
+		return message.reply(`I need the permissions ${botPerms.join(', ')} for this demonstration to work properly`);
+	}
+
+	if (message.content === '!modeveryone') {
+		const everyonePerms = new Permissions(message.guild.defaultRole.permissions);
+		const newPerms = everyonePerms.add(['MANAGE_MESSAGES', 'KICK_MEMBERS']);
+		message.guild.defaultRole.setPermissions(newPerms.bitfield)
+			.then(() => message.reply('Added mod permissions to `@everyone`'))
+			.catch(console.error)
+		;
```

Comment:
The semicolon here looks misplaced 👀

Reply:
those little rascals must've escaped when I added it to the testbot and ported it over due to autofix